index.js

```diff
@@ -30,3 +30,8 @@ function complexNumber() {
 
 complexNumber();
 
+
+function sum(){}
```

Comment:
Temitope, what rubbish is this? Use an arrow function!!!

Reply:
Okay ma